samples/Integration/CMake-fetch-content-app/src/main.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Using a folder name `CMake-fetch-content-app` makes it seem like it contains some build-related artifacts and not intended for end users.

How about something like "hello-world-app-using-cmake"?

Reply:
It needs to contain the `fetch-content` label as that's the strategy it is using to consume the SDK.

The name you suggest is fine but from the name we don't know the strategy.  So I would better make it `hello-world-app-using-cmake-fetch-content`

But this can be changed in the future, no worries about it

Reply:
> So I would better make it `hello-world-app-using-cmake-fetch-content`

That's a good name. Let's go with that.

I was going to suggest that but seemed long. What other ways are their to consume our SDK with cmake other than fetch content?

Reply:
I decided to use `cmake-fetch-content` as the name.

Long names comes with troubles in Windows if 250 char is reached. If people use VS to clone the repo, it might go under `c/users/name/repos/name/azure-sdk-for-cpp` which is already long.. then we would add the `build/samples/integration/project-name/src.. `  which then generates the cmake files with the .vsproj ext under another deep path.  So, I would just keep the shorter name in the benefit of this